yum/comps.py

```diff
@@ -28,11 +28,11 @@
 
 lang_attr = '{http://www.w3.org/XML/1998/namespace}lang'
 
-def parse_boolean(strng):
-    return BOOLEAN_STATES.get(strng.lower(), False)
+def parse_boolean(value):
+    return BOOLEAN_STATES.get(value.lower(), False)
 
-def parse_number(strng):
-    return int(strng)
+def parse_number(value):
+    return int(value)
 
```

Comment:
All of these are public APIs, Eg. yum.comps.parse_boolean(strng="foo") will now break after this change.